src/apis/status.ts

```diff
@@ -9,26 +58,91 @@ import chalk from 'chalk';
  * - Currently running local networks/nodes.
  * - Account information.
  * - Useful commands.
+ *
+ * @param env The name of the environment we are showing status for
  */
-export function cli() {
-  console.log(chalk.cyan('\n[Boson]'));
-  console.log('Version 0.1.0');
-  console.log('Backend: Geth 1.0.0 - Web3 1.0.0b3');
-
-  logProjectInfo();
-  logNetworkInfo();
-  console.log('\n');
-}
-
-function logProjectInfo() {
-  // TODO: Make this project aware (waiting for config).
-  if (true) {
-    console.log(chalk.cyan('\n[Project]'));
-    console.log('* Project: ' + chalk.bold('name') + '\n');
-    console.log('This folder has no double.yaml file.');
-  } else {
-    // console.log('This folder does not look like ');
+export async function cli(env: string, supressLogging?: boolean): Promise<IProjectStatus> {
+  const rendererConfig: {[index: string]: any} = {};
+  if (supressLogging) {
+    rendererConfig.renderer = require('listr-silent-renderer');
   }
+
+  const tasks = new Listr([
+    {
+      title: 'Reading Double configuration',
+      task: ctx => {
+        try {
+          Config.get();
+        } catch (err) {
+          throw new Error(`Cannot find any project configuration, please run boson init`);
+        }
+
+        try {
+          Config.getForEnv(env);
+        } catch (err) {
+          throw new Error(`Cannot find environment named ${env}, please check your configuration.`);
+        }
+
+        ctx.projConfig = Config.get();
+        ctx.envConfig = Config.getForEnv(env);
```

Comment:
also, this is a lot of dupe calls. you are calling both config.get and config.getforenv twice.